### DISCUSSION TITLE
Migration of attachments before or after an upgrade

### DISCUSSION THREAD

**Discussion by lukaspetr** [osTicket/osTicket#6755] [Q&A] [answered]
Dear colleagues, we would like to proceed with migration from db-based fs storage to filesystem storage. Currently, we have the latest 1.17 version of osTicket. Do you think it is better to first upgrade to 1.18 and then migrate or vice versa? Many thanks for any replies/notes!

**Comment by JediKev** [comment, marked as answer]
@lukaspetr 

Whatever way you want to do it; it will work both ways. An important note is that any time you do an upgrade on core osTicket make sure you have all the latest builds of any currently installed plugins specific to the version you are upgrading to (in this case v1.18.1) in your `include/plugins/` folder before running the upgrader in the UI. This way if the upgrader needs to upgrade something with the plugins they will be up-to-date and contain the needed information for said upgrade.

Cheers.

**Reply by lukaspetr** [reply]
Many thanks, @JediKev , this is helpful.
